src/display.py

```diff
@@ -211,6 +226,8 @@ def update(self):
 do_sorting = False
 paused = False
 timer_space_bar   = 0
+start_time = 0
+time_taken = 0
```

Comment:
I don't think `start_time` and `time_taken` need to be globals.
They are relevant only for `DisplayBox`, why not add them as class or instance variables?
There are several options for how to implement this. 
For example, you can set `start_time` when starting a new sort and update `time_taken` when calling the `update` method.
You can also save only `start_time` and directly do something like:
`self.text = str(time()- statr_time)[:6] + ' seconds'`